_posts/2018-11-23-biohackathon.md

```diff
@@ -1,7 +1,8 @@
 ---
 site: freiburg
-tags: [training]
+tags: [training, Bioconda, hackathon]
 title: ELIXIR Biohackathon in Paris
+supporter: ELIXIR
```

Comment:
I'm not sure if we should mix small and capital letters. Its easier for everyone if we keep all small?

Reply:
yes, I missed that one! Should be small caps...